REST Bindings/openapi.yaml

```diff
@@ -478,147 +476,15 @@ paths:
-        of EPCIS and CBV. The `OPTIONS` method allows the client to discover which vocabularies and EPCIS and CBV
-        versions are used.
-      responses:
-        '204':
-          description: |
-            The `OPTIONS` method is used as a discovery service for `events` endpoints. It describes which EPCIS and CBV
-            versions are supported as well as  EPCIS and CBV extensions.
-            The list of headers is not exhaustive. It only describes the functionality specific to EPCIS 2.0.
-          headers:
-            Allow:
-              schema:
-                example: OPTIONS, POST, GET
-                type: string
-                default: OPTIONS, POST, GET
-            GS1-EPCIS-Version:
-              $ref: '#/components/headers/GS1-EPCIS-Version'
-            GS1-EPCIS-Min:
-              $ref: '#/components/headers/GS1-EPCIS-Min'
-            GS1-EPCIS-Max:
-              $ref: '#/components/headers/GS1-EPCIS-Max'
-            GS1-CBV-Version:
-              $ref: '#/components/headers/GS1-CBV-Version'
-            GS1-EPC-Format:
-              $ref: '#/components/headers/GS1-EPC-Format'
-            GS1-CBV-XML-Format:
-              $ref: '#/components/headers/GS1-CBV-XML-Format'
-            GS1-EPCIS-Extensions:
-              $ref: '#/components/headers/GS1-EPCIS-Extensions'
-            GS1-Vendor-Version:
-              $ref: '#/components/headers/GS1-Vendor-Version'
-            GS1-CBV-Extensions:
-              $ref: '#/components/headers/GS1-CBV-Extensions'
-            GS1-EPCIS-Capture-Limit:
-              $ref: '#/components/headers/GS1-EPCIS-Capture-Limit'
-            GS1-EPCIS-Capture-File-Size-Limit:
-              $ref: '#/components/headers/GS1-EPCIS-Capture-File-Size-Limit'
-    get:
-      tags:
-        - 'Events'
-      summary: |
-        Returns EPCIS events of a given EPCIS event type. If `all` is used instead of an EPCIS
-        event type, every event in the repository is returned.
-      parameters:
-        - $ref: '#/components/parameters/PerPage'
-        - $ref: '#/components/parameters/NextPageToken'
-
-      description: |
-        This endpoint returns all EPCIS events of a specific EPCIS event type up to the amount defined in `perPage`.
-        The server returns a 'Link' header to point to the remaining results.
-
-        If the EPCIS event type is not needed, `all` can be used instead of an EPCIS event type.
-
-        The client can further restrict the list of events returned by filtering events using the EPCIS query
-        language as URL query string parameters.
-
-        Example:
-
-        ```
-        https://example.com/eventTypes/ObjectEvent/events?EQ_bizStep=urn%3Aepcglobal%3Acbv%3Abizstep%3Ashipping%2Curn%3Aepcglobal%3Acbv%3Abizstep%3Adecommissioning&GE_eventTime=2015-03-15T00%3A00%3A00.000-04%3A00
-        ```
+        This endpoint returns all EPCIS events that are currently in the repository. It supports
+        refinement via the EPCIS Query Language
```

Comment:
How about adding an example like below for refinement via the EPCIS Query Language?

```
        https://example.com/events?EQ_bizStep=urn%3Aepcglobal%3Acbv%3Abizstep%3Ashipping%2Curn%3Aepcglobal%3Acbv%3Abizstep%3Adecommissioning&GE_eventTime=2021-11-13T00%3A00%3A00.000-04%3A00
        ```

Reply:
yes, CBV compliant bare strings should be used in example.

Reply:
OK